networking.md

```diff
@@ -506,4 +506,166 @@ spec:
 Additionally there is a CRD-less `service_controller`. It watches for `LoadBalancer` services in the same namespace of a `DNSMasq` instance and creates a `DNSData` CR to auto register service endpoints which have an annotation set with  `"dnsmasq.network.openstack.org/hostname": <hostname>`. This annotation gets auto added to the `LoadBalancer` services exposed using lib-common `https://github.com/openstack-k8s-operators/lib-common/pull/258`.
 
 ## IPAM
-TBD
\ No newline at end of file
+The the infra-operator privides CRDs for IPAM.
+
+The `NetConfig` CRD allows do describe the overall networks used for the environment.
+
+* a `network` described by `name`, `mtu` (default 1500), and one or multiple subnets.
+* a `subnet` describedc by `name`, one or multiple `allocationRanges` with `start` and `end`, `cidr`, `vlan` id, `gateway` and `routes`. Using `excludeAddresses` individual IP addresses from within the `cidr` can be excluded to be assigned (dynamic or static).
```

Comment:
nit: typo s/describedc/described/

Generally the sentence is a bit hard to parse with comma and comma and. A suggestion that is a bit more words but possibly more readable. Alternatively we could use a "described by:" pattern and then bullet list explaining each field?

* a `subnet` described by: `name`, `cidr`, `vlan` (id), `gateway`, `routes`, `excludeAddresses` and `allocationRanges`. One or multiple `allocationRanges` can be defined, each with a `start` and `end` address. `routes` are optional, but one or multiple can be defined. Each route defining a `destination` network and `nexthop` router address. Using `excludeAddresses` individual IP addresses from within the `cidr` can be excluded to be assigned (dynamic or static).